### PR TITLE
Fix aspect to work with Bazel 5

### PR DESCRIPTION
**Pull request by guw**
Couple changes needed to discover what was going on. First exception stack traces are not logged in Eclipse. Once logged I noticed a scenario with my test project, which was not handled in BEF. Once that was fixed I could see the issue with aspects causing build issues.

### CODE REVIEW

**Comment by guw** [comment]
Note, the project is still not importing.
https://github.com/salesforce/bazel-jdt-java-toolchain

File: bundles/com.salesforce.bazel-java-sdk/aspect/intellij_info_impl.bzl [hunk 1]
```diff
@@ -971,23 +971,19 @@ def intellij_info_aspect_impl(target, ctx, semantics):
     direct_exports = []
     if JavaInfo in target:
         direct_exports = collect_targets_from_attrs(rule_attrs, ["exports"])
-        transitive_exports = target[JavaInfo].transitive_exports
-        export_deps = [
-            make_dep_from_label(label, COMPILE_TIME)
-            for label in transitive_exports.to_list()
-        ]
+        export_deps.extend(make_deps(direct_exports, COMPILE_TIME))
```
Comment (plaird) [inline bundles/com.salesforce.bazel-java-sdk/aspect/intellij_info_impl.bzl:974]: This change will get wiped out during the next copy of the Intelli aspect. We need to have a code comment here to show that this section has diverged intentionally from upstream (or did you get this fix from upstream?). Also in the note should explain the use case that requires the fix so we can retest after the next update.

Comment (guw) [reply bundles/com.salesforce.bazel-java-sdk/aspect/intellij_info_impl.bzl:974]: See the commit message:
https://github.com/salesforce/bazel-eclipse/pull/406/commits/6a3a74a1724e9792f26a9a6d1fc71cfea2383a90

Comment (plaird) [reply bundles/com.salesforce.bazel-java-sdk/aspect/intellij_info_impl.bzl:974]: OK great.

**Comment by plaird** [comment]
OK, here are some related items:

#404 may apply because your git repo has a BUILD file at root. But probably not for this initial import problem, but something be aware of.

W-10767740 (internal sfdc work item) which is not public yet because we don't have a diagnosis yet and the affected project is not public

**Comment by plaird** [comment]
Another thing to try is in Bazel Preferences for your Eclipse workspace. There is a checkbox "Enable project structure optimizations during import" which is enabled by default. Disable that.

**Pull request merged**